### PR TITLE
gettext wrapper for title of ToolBarButton

### PR DESCRIPTION
**Pull request by berkuteagle**
Added a third parameter to the Sequeler.Partials.ToolBarButton constructor for the localized title

### CODE REVIEW

File: src/Partials/ButtonType.vala [hunk 1]
```diff
@@ -41,12 +41,12 @@ namespace Sequeler.Partials {
...
 
 	public class ToolBarButton : Gtk.Grid {
-		public ToolBarButton (string icon_name, string tooltip) {
+		public ToolBarButton (string icon_name, string tooltip, string label) {
 			name = tooltip;
 			halign = Gtk.Align.CENTER;
 
 			Gtk.Image icon;
-			var title = new Gtk.Label (tooltip);
+			var title = new Gtk.Label (label);
```
Comment (Alecaddd) [inline src/Partials/ButtonType.vala]: Why doing this and leaving the tooltip untranslatable? Wouldn't be better to simply add the translations for the tooltip string, used in both title and tooltip name?

Comment (berkuteagle) [reply src/Partials/ButtonType.vala]: In https://github.com/Alecaddd/sequeler/blob/master/src/Layouts/DataBaseView.vala#L61 you check the selected tab by name.
I think it's not very good to use the translated string as a name.

Comment (berkuteagle) [reply src/Partials/ButtonType.vala]: It seems right now tabs do not work in non-English locale

Comment (Alecaddd) [reply src/Partials/ButtonType.vala]: Ah, damn, I'm an idiot.
You're right, and I forgot how my own application works :laughing:

**Pull request merged**